### ISSUE TITLE
Demo site 403 error

### ISSUE THREAD

**Issue by ghost** [mattosaurus/ChartJSCore#89]
Hello,

Demo site is not working. 
This project can be seen in action on the demo site [here](https://bitscry.com/Projects/Chart).

Br,
Saulius

**Comment by mattosaurus** [comment]
Yeah, I need to sort out the hosting for this site. Thanks for reminding me.

I'll remove it from the docs for now.

**Comment by ghost** [comment]
Great, thanks

<https://nobeds.com/>

NOBEDS.COM <http://nobeds.com/> - Free hotel management system

*Support team* - Saulius Chomentauskas

PO BOX 30, H. Manto 7, Klaipėda

LT-91247, Lithuania, +370 611 54 444

***@***.***


<https://www.youtube.com/channel/UCMa4zsrXPUDjiS688dP5dYw?sub_confirmation=1>



This e-mail may contain confidential and/or privileged information. If you
are not the intended recipient please notify the sender immediately and
destroy this e-mail. Any unauthorized copying, disclosure or distribution
of the material in this e-mail is strictly forbidden.
